collector/benchmarks/README.md

```diff
@@ -26,7 +26,6 @@ They mostly consist of real-world crates.
   trait bounds, by having a lot of trait impls for a large number of different
   types.
 - **encoding**: Character encoding support. Contains some large tables.
-- **futures**: A futures implementation. Used by many Rust programs.
```

Comment:
Do we want to add one of the subcrates from futures as a replacement? 

`futures-core` has 1.9mil downloads which is the most of any crate in the `futures` crate ecosystem. It's not a super interesting crate though. `futures-io` is also popular (albeit a bit less than `futures-core`), and it's perhaps a bit more interesting (e.g., heavy usage of `macro_rules!`, some semi complicated blanket trait impls, etc.).

Reply:
I don't think so, because these sub-crates are all quite small. I have some thoughts about other, more popular crates to add, e.g. `log`, `serde_derive`, `libc`, `proc-macro2`, `quote`, `memchr`, `tokio`.